remarkable-review/The End of Paper.md

```diff
@@ -0,0 +1,101 @@
+# The End of Paper
+<picture of my moleskin collection>
+Over the last thirteen years, I've filled twenty-six Moleskines with daily notes, reflections, and scribblings. Within a few minutes of laying my hands on the reMarkable tablet, I knew I'd never buy another paper notebook. It's that good.
+
+How was I convinced to replace a decade-old allegiance to analog in under ten minutes? Well, let's just say the reMarkable tablet is aptly named.
+
+I'll walk you through my paper-based workflows, the various digital products I've tried to replace them with, and then my reMarkable experience.
+
+## My love affair with paper
+<photo of my stacks of moleskines>
+The first dated entry I could find from one of my notebooks is X/X/2000. The writings began as the passionate overwrought ramblings of a high schooler. During [undergrad](http://cs.trinity.edu/) and [grad school](https://cs.unc.edu/), I wrote haphazardly, not really establishing a structured habit for my usage of paper.
+
+Beginning in 2007, I got serious. I carried a paper notebook with me everywhere. And I mean everywhere. I started my day by cracking it open, carried it to all my meetings, lunches, church, travel, everywhere. Notebook-carrying became such a fixture of my identity that when I got married in 2010 the write-everything-down habit was hilariously caricatured by a friend using a 3 foot by 4 foot cardboard notebook, acting out me writing down every word while on a date with my wife.
+
+<photo of Eric with a "paper notebook">
+
+I wrote nearly every day. I also evolved my system continually. I moved from a single notebook for capturing everything to two: one for a daily working log and one for personal, long-form journaling. I experimented with different formatting techniques for pages. I dabbled in a variety of paper todo systems. I experimented with different notebook form factors: soft cover, hard cover, small, medium, large, lined, unlined, grid. I scribbled my way through sheafs of dead tree matter. Writing even seem to reshape my brain: my habit of thought defaulted to externalizing ideas on paper or whiteboard.
+
+And I loved it.
+
+I loved starting a new day with a fresh page, noting the day's date, and opening myself to the possibilities in front of me. I loved dedicating time to look back on life, sorting through emotions, churning thought-trains, and half-made plans. From brain to hand to pen, the ink flowing onto the page created a black-and-beige clarity that seemed more difficult when staring at a glowing rectangle or typing on my keyboard.
+
+All that's to say, I was and am still deeply in love with paper. 
+
+## A dalliance with digital
+As wonderful as paper had been to me, as a digital native, I was constantly aware of the tradeoff I was making: digital has its advantages.
```

Comment:
To me, this opening indicates that you're about to discard paper, which hasn't happened yet in the storyline. I'd open with something that sounds more like you're about to unfold a concurrent storyline (which you are, since the next paragraph jumps back to 2003). I'm thinking something more like "Perhaps unlike many notebook aficionados, I have always been a digital native. So while the evolution of my pen and paper system was extensive, it isn't the full story."

Reply:
Great point. Much better opener.